pkg/resource/init_metadatas.go

```diff
@@ -0,0 +1,29 @@
+package resource
+
+import (
+	"github.com/pkg/errors"
+	"github.com/snyk/driftctl/enumeration/remote/common"
+	"github.com/snyk/driftctl/enumeration/resource"
+	"github.com/snyk/driftctl/pkg/resource/aws"
+	"github.com/snyk/driftctl/pkg/resource/azurerm"
+	"github.com/snyk/driftctl/pkg/resource/github"
+	"github.com/snyk/driftctl/pkg/resource/google"
+)
+
+func InitMetadatas(remote string,
```

Comment:
This is already done in `remote.Activate()`, is that really usefull to call that manually ?

Reply:
this is not the same init. This one will add normalize function that only exists in driftctl so it cannot be call from the enumeration function